contrib/examples/actions/mistral-handle-retry.yaml

```diff
@@ -0,0 +1,30 @@
+version: '2.0'
+name: "examples.mistral-handle-retry"
+
+workflows:
+
+    main:
+        type: direct
+        tasks:
+            task1:
+                workflow: work
+                policies:
+                    retry:
+                        count: 20
+                        delay: 3
```

Comment:
i tried break-on.  the thing is the retry policy don't have an option to run an undo.